microsoft/atlas-design
# Update danger color

Link: [preview](http://localhost:1111/)

This PR updates the danger semantic color to match Fluent 2 on light theme.

## Testing

1. Visit the following pages to review the affected colors, should still work on dark and high contrast theme. Compare with https://react.fluentui.dev/ . Also compare with https://design.learn.microsoft.com/ for the current Atlas colors.

- [Color | Atlas Design | Microsoft](http://localhost:1111/atomics/colors.html)
- [Badge | Atlas Design | Microsoft](http://localhost:1111/components/badge.html)
- [Button | Atlas Design | Microsoft](http://localhost:1111/components/button.html)
- [Checkbox | Atlas Design | Microsoft](http://localhost:1111/components/checkbox.html)
- [Form | Atlas Design | Microsoft](http://localhost:1111/components/form.html)
    - submit a form to generate validation error
    - Color change also affects asterisk/required indicator color
- [Help | Atlas Design | Microsoft](http://localhost:1111/components/help.html)
- [Input | Atlas Design | Microsoft](http://localhost:1111/components/input.html)
- [Notification | Atlas Design | Microsoft](http://localhost:1111/components/notification.html)
    - https://react.fluentui.dev/?path=/docs/components-messagebar--docs
- [Progress Bar | Atlas Design | Microsoft](http://localhost:1111/components/progress-bar.html)
- [Select | Atlas Design | Microsoft](http://localhost:1111/components/select.html)
- [Text Area | Atlas Design | Microsoft](http://localhost:1111/components/textarea.html)

## Additional information

[Optional]

## Contributor checklist
